classes/patterns/class-issues.php

```diff
@@ -53,52 +55,36 @@ public static function get_config( $params = [] ): array {
 			'title'      => __( 'Issues', 'planet4-blocks-backend' ),
 			'categories' => [ 'planet4' ],
 			'content'    => '
-				<!-- wp:group {"className":"' . $classname . '","align":"full","backgroundColor":"grey-05"} -->
-				<div class="wp-block-group ' . $classname . ' alignfull has-grey-05-background-color has-background">
+				<!-- wp:group {"className":"' . $classname . '","align":"full","backgroundColor":"grey-05","style":{"spacing":{"padding":{"top":"80px","right":"0px","bottom":"80px","left":"0px"}}}} -->
```

Comment:
Do you need to specify the `0px` ones? 🤔 I see that for the heading you only put the margin bottom and it seems to work, it just leaves the other fields empty in the sidebar but I don't think that's an issue 🤷‍♀️
<img width="278" alt="Screenshot 2022-08-04 at 17 34 56" src="https://user-images.githubusercontent.com/6949075/182887710-79885e50-c32c-4060-8cf2-4a199800d559.png">

Reply:
Not necessary! In fact, if you don't set values, those values won't be added to styles.
If you set `0px 0px 24px 0px`it will be rendered as `{"margin-top: "0px","margin-right: "0px","margin-bottom "24px","margin-left "0px"}`

However, if you set only 
If you set `margin-bottom` to `24px`it will be rendered as `{"margin-bottom": "24px"}`

Reply:
Exactly my point 🙂 Maybe we can remove them then? These patterns are already quite complex so if we can remove avoid adding extra stuff I think it's better!